qa/L0_backend_python/env/test.sh

```diff
@@ -123,6 +123,42 @@ for EXPECTED_VERSION_STRING in "$PY36_VERSION_STRING" "$PY37_VERSION_STRING" "$P
         RET=1
     fi
 done
+grep "Locale is \(None, None\)" $SERVER_LOG
```

Comment:
Instead of adding a comment in the CMake file per my original comment, can you instead add some brief description for why we're testing this here?

Something like this:
```
# Test locale in python stub processes
# NOTE: In certain pybind versions, the locale settings may not be propagated from parent to
#       stub processes correctly. See https://github.com/triton-inference-server/python_backend/pull/260.
```